sale_order_line_sequence/views/report_saleorder.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
Please add license/copyright

Reply:
@alexey-pelykh It's not a mandatory one in XML, no need here

Reply:
Absolutely true, that's why not a request change yet just a comment. Still a code, of some sort, still licensed somehow